README.md

```diff
@@ -1,190 +1,21 @@
 <div align="center">
-  <img alt="Fastly CLI logo" src="https://raw.githubusercontent.com/fastly/cli/master/docs/img/cli-logo.svg?sanitize=true" height="140" />
```

Comment:
Why are we loosing the illustration 😢

Reply:
I knew this would upset someone 🤦🏻 

IMO it looks out of place.

I also don't think this particular project benefits from having a 'mascot' or logo.